docs/docs/reference/other-new-features/multiversal-equality.md

```diff
@@ -9,112 +9,150 @@ the fact that `==` and `!=` are implemented in terms of Java's
+```scala
+implied for Eql[T, T] = Eql.derived
+```
+This definition effectively says that values of type `T` can (only) be
+compared to other values of type `T` when using `==` or `!=`. The definition
+affects type checking but it has no significance for runtime
 behavior, since `==` always maps to `equals` and `!=` always maps to
-the negation of `equals`. The right hand side of the definition is a value
-that has any `Eq` instance as its type. Here is the definition of class
-`Eq` and its companion object:
-
+the negation of `equals`. The right hand side `Eql.derived` of the definition
+is a value that has any `Eql` instance as its type. Here is the definition of class
+`Eql` and its companion object:
 ```scala
 package scala
 import annotation.implicitNotFound
 
 @implicitNotFound("Values of types ${L} and ${R} cannot be compared with == or !=")
-sealed trait Eq[-L, -R]
+sealed trait Eql[-L, -R]
 
-object Eq extends Eq[Any, Any]
+object Eql {
+  object derived extends Eql[Any, Any]
+}
 ```
 
-One can have several `Eq` instances for a type. For example, the four
+One can have several `Eql` instances for a type. For example, the four
 definitions below make values of type `A` and type `B` comparable with
 each other, but not comparable to anything else:
 
 ```scala
-implicit def eqA : Eq[A, A] = Eq.derived
-implicit def eqB : Eq[B, B] = Eq.derived
-implicit def eqAB: Eq[A, B] = Eq.derived
-implicit def eqBA: Eq[B, A] = Eq.derived
+implied for Eql[A, A] = Eql.derived
+implied for Eql[B, B] = Eql.derived
+implied for Eql[A, B] = Eql.derived
+implied for Eql[B, A] = Eql.derived
 ```
+The `scala.Eql` object defines a number of `Eql` instances that together
+define a rule book for what standard types can be compared (more details below).
 
-(As usual, the names of the implicit definitions don't matter, we have
-chosen `eqA`, ..., `eqBA` only for illustration).
-
-The `scala.Eq` object defines a number of `Eq` implicits that make
-values of types `String`, `Boolean` and `Unit` only comparable to
-values of the same type. They also make numbers only comparable to
-other numbers, sequences only comparable to other
-sequences and sets only comparable to other sets.
-
-There's also a "fallback" instance named `eqAny` that allows comparisons
-over all types that do not themselves have an `Eq` instance.  `eqAny` is
+There's also a "fallback" instance named `eqlAny` that allows comparisons
+over all types that do not themselves have an `Eql` instance.  `eqlAny` is
 defined as follows:
 
 ```scala
-def eqAny[L, R]: Eq[L, R] = Eq.derived
+def eqlAny[L, R]: Eql[L, R] = Eql.derived
 ```
 
-Even though `eqAny` is not declared implicit, the compiler will still
-construct an `eqAny` instance as answer to an implicit search for the
-type `Eq[L, R]`, provided that neither `L` nor `R` have `Eq` instances
-defined on them.
+Even though `eqlAny` is not declared `implied`, the compiler will still
+construct an `eqlAny` instance as answer to an implicit search for the
```

Comment:
Is there also an alternative name for *implicit search*? Mixing the two terminologies here makes the sentence a bit confusing...